core/src/main/scala/kafka/tools/StreamsResetter.java

```diff
@@ -157,49 +164,63 @@ private void parseArguments(final String[] args) throws IOException {
         }
     }
 
-    private void resetInputAndInternalAndSeekToEndIntermediateTopicOffsets() {
+    private void mayberesetInputAndInternalAndSeekToEndIntermediateTopicOffsets() {
         final List<String> inputTopics = options.valuesOf(inputTopicsOption);
         final List<String> intermediateTopics = options.valuesOf(intermediateTopicsOption);
 
-        if (inputTopics.size() == 0 && intermediateTopics.size() == 0) {
-            System.out.println("No input or intermediate topics specified. Skipping seek.");
-            return;
-        } else {
-            if (inputTopics.size() != 0) {
-                System.out.println("Resetting offsets to zero for input topics " + inputTopics + " and all internal topics.");
-            }
-            if (intermediateTopics.size() != 0) {
-                System.out.println("Seek-to-end for intermediate topics " + intermediateTopics);
+        final List<String> internalTopics = new ArrayList<>();
+
+        final List<String> notFoundInputTopics = new ArrayList<>();
+        final List<String> notFoundIntermediateTopics = new ArrayList<>();
+
+        Boolean dryRun = options.has(dryRunOption);
+        String groupId = options.valueOf(applicationIdOption);
+
+        if (!dryRun) {
+            if (inputTopics.size() == 0 && intermediateTopics.size() == 0) {
+                System.out.println("No input or intermediate topics specified. Skipping seek.");
+                return;
+            } else {
+                if (inputTopics.size() != 0) {
+                    System.out.println("Resetting offsets to zero for input topics " + inputTopics + " and all internal topics.");
+                }
+                if (intermediateTopics.size() != 0) {
+                    System.out.println("Seek-to-end for intermediate topics " + intermediateTopics);
+                }
             }
+        } else {
+            System.out.println("----Dry run displays the actions which will be performed when running Streams Reset Tool----");
         }
 
-        final Properties config = new Properties();
-        config.putAll(consumerConfig);
-        config.setProperty(ConsumerConfig.BOOTSTRAP_SERVERS_CONFIG, options.valueOf(bootstrapServerOption));
-        config.setProperty(ConsumerConfig.GROUP_ID_CONFIG, options.valueOf(applicationIdOption));
-        config.setProperty(ConsumerConfig.ENABLE_AUTO_COMMIT_CONFIG, "false");
-
         final Set<String> topicsToSubscribe = new HashSet<>(inputTopics.size() + intermediateTopics.size());
+
         for (final String topic : inputTopics) {
             if (!allTopics.contains(topic)) {
-                System.err.println("Input topic " + topic + " not found. Skipping.");
+                notFoundInputTopics.add(topic);
```

Comment:
This is done, to make output clear 
System.out.println("----Dry run displays the actions which will be performed when running Streams Reset Tool----");

As I am printing this, and below printing all dry-run actions, so want t make it clear with description done like this.
Following input topics are not found, skipping them
Topic: t1
Following input topics are not found, skipping them
Topic:t2